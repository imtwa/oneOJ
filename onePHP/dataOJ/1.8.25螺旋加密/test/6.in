12 12 AZUBLSDUCASHAXVCYPOKILJQYGD