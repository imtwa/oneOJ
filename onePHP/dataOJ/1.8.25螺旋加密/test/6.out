000011101010110000110011011100110010011000101001001111000010100000011000001000000100100100000101010001110100111011010100001000111001010011001001